markets/perps-market/contracts/modules/AccountModule.sol

```diff
@@ -14,4 +16,17 @@ contract AccountModule is IAccountModule {
     function totalAccountOpenInterest(uint128 accountId) external view override returns (int) {
         return PerpsAccount.load(accountId).getTotalAccountOpenInterest(accountId);
     }
+
+    function openPosition(
```

Comment:
should we rename this to `getAccountPosition` or `getOpenPosition`?  `openPosition` sounds like we're opening a position.

Reply:
yea I wasn't sure what we want for views.. I went with no `get` prefix but we could add it for all view functions